apps/api/v2/src/app.rewrites.middleware.ts

```diff
@@ -10,6 +10,9 @@ export class RewriterMiddleware implements NestMiddleware {
     if (req.url.startsWith("/v2/ee")) {
       req.url = req.url.replace("/v2/ee", "/v2");
     }
+    if (req.url.contains("reccuring")) {
```

Comment:
Are people already using `reccuring`?

Reply:
it's possible but probably nobody is since we didnt expose the hook to create recurring booking